lib/heroku/command/pg.rb

```diff
@@ -14,6 +14,16 @@ def set_commands(shorthand)
     end
   end
 
+  # pg:stats_reset
+  # calls the Postgres functions pg_stat_reset()
```

Comment:
Ok, so the two different things are `pg_stat_statements_reset` and `pg_stat_reset`.

The former resets stats the system has been collecting about what queries you are running: basically everything visible in the [`pg_stat_statements`](http://www.postgresql.org/docs/9.4/static/pgstatstatements.html) view: the most-frequently-run queries, how many times they've been run, the average execution time, et cetera. These are very useful to tune your application, but it's not directly used by Postgres at all.

The latter resets stats the system uses about what your data looks like. Postgres has a cost-based query planner, which means it looks at your query, looks at your schema, looks at what it knows about your data [1], considers the different ways it could execute your query, and picks the cheapest one. What Postgres knows about your data is visible via the [`pg_stats`](http://www.postgresql.org/docs/9.4/static/view-pg-stats.html) view, and is gathered by the [`ANALYZE`](http://www.postgresql.org/docs/9.4/static/sql-analyze.html) command (either directly by the user, or by the autovacuum daemon as part of ensuring your database is vacuumed). The way Postgres figures out which plan is cheapest depends largely on the [planner settings and cost constants](http://www.postgresql.org/docs/9.4/static/runtime-config-query.html). Resetting stats is rarely useful (instead, it's usually better to gather new, more up-to-date stats to replace the old ones by running `ANALYZE` again), but it can be handy for testing and after drastic changes to your data distribution.

I think it'd be worthwhile for you to come up with whatever docs you feel are appropriate and work with Rimas or whoever else is interested to make sure they work. I'm happy to help! Let me know if you have more questions.

[1]: Which can affect query planning a lot: for a trivial example, consider a table like `a(b integer)` with an index on `b` and a query like `select * from a where b > 10`. If the table has ten records, the optimal plan is to scan the whole table and *then* filter. If the table has a million records, it's best to go to the index, find matching records, and then look up each record.